prow/plugins/config.go

```diff
@@ -429,6 +429,9 @@ type AssignOnLabel struct {
 type Trigger struct {
 	// Repos is either of the form org/repos or just org.
 	Repos []string `json:"repos,omitempty"`
+	// TrustedUsers is the explicit list of GitHub users whose PRs will be automatically
+	// considered as trusted. By default, trigger will ignore this list.
+	TrustedUsers []string `json:"trusted_users,omitempty"`
```

Comment:
Trust happens implicitly by being a collaborator on the repo or member of the repos org. The only use case I am aware of where this doesn't work are GitHub apps, ref https://github.com/kubernetes/test-infra/issues/22731

Is that your usecase? If so, can we rename the field to clarify that it is for apps only and I believe we also need to append an `[app]` to the name then?

Reply:
I'm not a fan of the idea of making this a `trused_users` setting rather than a `trusted_apps` one, because:
* It overlaps with trusting users by adding them to the org or repo and will make it difficult to clarify when to use which
* It is hard to discover that this is to be used when someone wants to trust an app

Reply:
@alvaroaleman You are right. I've edited the code to reflect this setting should address only GitHub apps that end with either `[app]` or `[bot]` suffix.